src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIBasic.java

```diff
@@ -245,8 +253,13 @@ private IAIState getNeededItem()
                 setDelay(2);
                 return getState();
             }
-
-            tryTransferFromPosToWorker(walkTo, needsCurrently);
+            // modified for debug and added the Stack itself as a workaround to get the "real Stacksize"
+            if(tryTransferFromPosToWorker(walkTo, needsCurrently))
```

Comment:
If this was done for debug, and it does nothing, then please remove it for live. 
No need to keep it

Reply:
comment line needs to be changed

Reply:
This should still be removed (nothing inside the if/else cases)

Reply:
already modified